src/main/java/carbonconfiglib/networking/snyc/SyncPacket.java

```diff
@@ -93,7 +93,7 @@ public void read(FriendlyByteBuf buffer) {
 	public void process(Player player) {
 		ReloadMode mode = processEntry(player);
 		if(mode != null) {
-			player.sendSystemMessage(mode.getMessage());
+			player.sendMessage(mode.getMessage(), player.getUUID());
```

Comment:
Please Util.NillUUID or whatever it is called instead. No sender provided, since it is a systems message